network/requirements.txt

```diff
@@ -1,5 +1,5 @@
-tensorflow==1.15
-tensorflow-probability==0.11.0
+tensorflow-gpu==1.15
+tensorflow-probability==0.7.0
```

Comment:
Zmieniłam tf na 1.15, musiałam też zrobić downgrade tfp, bo 0,11,0 nie było kompatybilne

Reply:
A próbowałaś z `tf2.0`?
I jakieś tf probability pod to

Reply:
Na tf 2.* jest zmienione api, dlatego nie działały rzeczy